tools/pdf2txt.py

```diff
@@ -18,7 +18,7 @@
 
 def float_or_disabled(x):
     if x.lower().strip() == "disabled":
-        return x
+        return "disabled"
```

Comment:
Based on the line above, the intended behaviour appears to have been that "Disabled" etc. is permitted; in that case makes sense to normalise the various cases to a single string.